doc/kinsol/guide/source/Usage/index.rst

```diff
@@ -2126,6 +2139,45 @@ of type :c:type:`KINDampingFn` to computing the damping factor, :math:`\beta_n`.
      a non-zero value if an error occurred.
 
 
+.. _KINSOL.Usage.CC.user_fct_sim.depthFn:
+
+Depth function
+~~~~~~~~~~~~~~
+
+When using the fixed-point or Picard iterations, the user may provide a function
+of type :c:type:`KINDepthFn` to modify the depth, :math:`m_n`.
+
+.. c:type:: int (*KINDepthFn)(long int iter, N_Vector u_val, N_Vector g_val, N_Vector f_val, N_Vector* df, sunrealtype* R_mat, long int depth, void* user_data, long int* new_depth, long int* remove_indices);
+
+   This function computes the depth parameter, :math:`m_{\textrm{max}} \leq m_n
+   \geq 0`, for fixed-point and Picard iterations.
```

Comment:
Are these inequalities correct?  

Also, the wording of this makes it difficult to determine if the function returns `m_n` or `m_{\textrm{max}}`.  In the text below, it looks like it actually returns `:math:`\hat{m}_n` -- I think that should be listed here for clarity.

Reply:
> Are these inequalities correct?

Nope, ends are reversed.

> Also, the wording of this makes it difficult to determine if the function returns `m_n` or `m_{\textrm{max}}`. In the text below, it looks like it actually returns `:math:`\hat{m}_n` -- I think that should be listed here for clarity.

Added some more detail to make this clearer.